apps/ledger-live-mobile/src/components/AccountGraphCard.tsx

```diff
@@ -47,9 +48,10 @@ const Footer = ({ renderAccountSummary }: FooterProps) => {
   return accountSummary ? (
     <Box
       flexDirection={"row"}
-      alignItemps={"center"}
+      alignItems={"center"}
       marginTop={5}
       overflow={"hidden"}
+      background="green"
```

Comment:
What is this `green` ?

Reply:
Forgotten debug thing. Just got rid of it